scripts/lib/aws.sh

```diff
@@ -15,7 +15,7 @@ ensure_ecr_repo() {
 }
 
 ensure_aws_k8s_tester() {
-    TESTER_RELEASE=${TESTER_RELEASE:-v1.0.0}
```

Comment:
Lots of new stuff!

https://github.com/aws/aws-k8s-tester/blob/master/CHANGELOG/CHANGELOG-1.1.md